src/main/java/com/algorand/algosdk/crypto/Address.java

```diff
@@ -110,6 +128,38 @@ public String encodeAsString() throws NoSuchAlgorithmException {
         return res;
     }
 
+    /**
+     * verifyBytes verifies that the signature for the message is valid for the public key
```

Comment:
Also worth documenting the MX prefix here, lest users get confused and try to verify generic signatures